root/static/scripts/edit/utility/guessFeat.js

```diff
@@ -241,7 +241,7 @@ export default function guessFeat(entity) {
     names: artistCredit.concat(match.artistCredit),
   });
 
-  entity.artistCreditEditorInst && entity.artistCreditEditorInst.setState({
+  entity.artistCreditEditorInst?.current?.setState({
```

Comment:
I didn't find docs for the "current" thing, what's the change? Or did you just decide to do this to simplify stuff?

Reply:
Those are refs: https://reactjs.org/docs/refs-and-the-dom.html (they're not DOM specific though, you can store any value in them).

A ref is just an object with a single property, `current`, that holds some value.  The idea is that the actual ref doesn't change (`entity.artistCreditEditorInst` stays constant), but the value stored in `current` can change.

Reply:
We used to get the component instance like `entity.artistCreditEditorInst = ReactDOM.render(...)`, which is no longer supported in the new API.  Now we basically do this:

```js
entity.artistCreditEditorInst = React.createRef();

// ...

<ArtistCreditEditor ref={entity.artistCreditEditorInst} {...props} />
```

Reply:
Oh, great, I didn't follow that the artistCreditEditorInst thing was being created differently on a second file :)